CHANGELOG.md

```diff
@@ -0,0 +1,245 @@
+# Changelog
+
+## [Unreleased](https://github.com/unit8co/darts/tree/develop)
```

Comment:
I think we can already put a reference to a new version here so that it comes out with the release